pkg/parsing/impl/validator_test.go

```diff
@@ -896,8 +896,8 @@ func newParser(t *testing.T, prefixes []string, opts ...parsing.Option) parsing.
 }
 
 // Helpers to have a pointer to pointer for generic test-case running.
-func ptr2ErrInvalidSyntax() **parsing.ErrInvalidSyntax {
-	var e *parsing.ErrInvalidSyntax
+func ptr2ErrInvalidSyntax() **sqlparser.ErrSyntaxError {
```

Comment:
Some tests check for errors in syntax, so this change was necessary so tests could still pass.